gpumon/gpumon_interface.ml

```diff
@@ -72,11 +74,13 @@ type gpu_errors =
 exception Gpumon_error of gpu_errors
 
 (** Error handler *)
-module GpuErrors = Error.Make(struct
-    type t = gpu_errors
-    let t = gpu_errors
-  end)
-let gpu_err = GpuErrors.error
+let gpu_err = Error.{
+    def = gpu_errors;
+    raiser = (function | e -> raise (Gpumon_error e));
```

Comment:
Here `fun e -> raise (Gpumon_error e)` would be more natural as we don't have several patterns.

Reply:
Indeed, I went with copy-paste. I'll add fixup commits